cmd/common/template/usage_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package template
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/cilium/hubble/pkg/defaults"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUsage(t *testing.T) {
+	cmd := &cobra.Command{
+		Use:     "cmd",
+		Aliases: []string{"and", "conquer"},
```

Comment:
nice!

Reply:
I was expecting you to catch this one :)